docker-compose.yml

```diff
@@ -1,37 +1,26 @@
---- 
-networks: 
-  firefly_iii_net: 
-    driver: bridge
-services: 
-  firefly_iii_app: 
+version: '3.3'
+
+services:
+  firefly_iii:
     image: jc5x/firefly-iii:latest
+    volumes:
+      - firefly_iii_export:/var/www/firefly-iii/storage/export
+      - firefly_iii_upload:/var/www/firefly-iii/storage/upload
+    env_file: .env
+    ports:
+      - 8080:80
```

Comment:
Could you restore this to port 80?

Reply:
Sure, i just used 8080 for testing on my local machine

Reply:
Should be changed now :)